.github/workflows/release.yml

```diff
@@ -40,18 +43,24 @@ jobs:
       - if: steps.tag_exists.outputs.exists == 'true'
         run: exit 1
 
-      #########################################################################
-      # Relevant build steps here
-      #########################################################################
+      # Publish the release to our package manager
+      - uses: ./.github/actions/publish-package
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          name: ${{ steps.get_version.outputs.version }}
+          body: ${{ github.event.pull_request.body }}
+          tag: ${{ steps.get_version.outputs.version }}
+          commit: ${{ github.sha }}
```

Comment:
Mostly informational, but if we have a PR with multiple commits, and we squash it in the release PR, would github.sha point to the new commit on the target branch (e.g. main / beta) ?

Reply:
Correct! This workflow is triggered after a release PR is merged, so the sha reflects the just-merged-commit to that target branch.